packages/frontend/src/components/ProjectTablesConfiguration/ProjectTablesConfiguration.tsx

```diff
@@ -156,119 +158,124 @@ const ProjectTablesConfiguration: FC<{
-                            by commas.
-                        </TextP>
-                        {typeValue === TableSelectionType.WITH_TAGS && (
-                            <MultiSelect
-                                name="tags"
-                                label="Tags"
-                                rules={{
-                                    required: 'Required field',
-                                }}
-                                items={availableTags}
-                                disabled={disabled}
-                                placeholder="e.g lightdash, prod"
+                        <Stack mt={'md'} spacing={'md'}>
+                            <Radio
+                                value={TableSelectionType.ALL}
+                                label="Show entire project"
+                                description="Show all of the models in your dbt project in Lightdash."
                             />
-                        )}
-
-                        <Radio
-                            label="Show models in this list"
-                            value={TableSelectionType.WITH_NAMES}
-                        />
-                        <TextP>
-                            Write a list of models you want to include,
-                            separated by commas.
-                        </TextP>
-                        {typeValue === TableSelectionType.WITH_NAMES && (
-                            <MultiSelect
-                                name="names"
-                                label="Names"
-                                rules={{
-                                    required: 'Required field',
-                                }}
-                                items={(explores || []).map(({ name }) => name)}
-                                disabled={disabled}
-                                placeholder="e.g users, orders"
-                            />
-                        )}
-                    </RadioGroup>
+                            <Box>
+                                <Radio
+                                    value={TableSelectionType.WITH_TAGS}
+                                    label={
+                                        <>
+                                            Show models with any of these tags{' '}
+                                            <DocumentationHelpButton href="https://docs.getdbt.com/reference/resource-configs/tags#examples" />
+                                        </>
+                                    }
+                                    description="Write a list of tags you want to include, separated
+                                    by commas."
+                                />
+                                {form.values.type ===
+                                    TableSelectionType.WITH_TAGS && (
+                                    <MultiSelect
```

Comment:
At the moment if there aren't any tags and you select this, it's marked as required, but you can't input anything there. Maybe if there aren't any it should be disabled? The old behavior is about the same, but its still bad.

Or do we expect people to be able to add tags here that aren't already defined? In which case maybe we need the tag input.

Reply:
Mantine tooltip doesn't work when wrapping a disabled input 😢 Used the error message instead, even though it doesn't stop them from saving but that seems minor. 
<img width="882" alt="Screenshot 2023-12-12 at 17 11 33" src="https://github.com/lightdash/lightdash/assets/9117144/464ea386-b880-4600-b683-c386b485f2c7">

Reply:
you can put tag input inside a `div` (tooltip > div > taginput)
btw, it's the same for all elements in react. if input is disabled it does not trigger any event from the pointer